### DISCUSSION TITLE
[Feature] Remote Machine Learning URL fallback to host/elsewhere when remote is down

### DISCUSSION THREAD

**Discussion by saildot4k** [immich-app/immich#12808] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

ISSUE:
When a remote machine learning container is setup, and the container is down, smart search does not work anymore.

REQUEST
Allow setup of fallback to localhost such that at least smart search still works. 

User u/cof666 on reddit posted such a  script [fix](https://www.reddit.com/r/immich/comments/1faif9e/smart_search_not_working_possible_solutions/): 

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Dupe: #4616
